src/main.cpp

```diff
@@ -811,11 +820,22 @@ int main(int argc, char**argv)
     diameter_resolver = new DiameterResolver(dns_resolver,
                                              af,
                                              options.diameter_blacklist_duration);
-    realm_manager = new RealmManager(diameter_stack,
-                                     options.dest_realm,
-                                     options.dest_host,
-                                     options.max_peers,
-                                     diameter_resolver);
+    if (options.force_hss_peer.empty())
+    {
+      realm_manager = new RealmManager(diameter_stack,
+                                       options.dest_realm,
+                                       options.dest_host,
+                                       options.max_peers,
+                                       diameter_resolver);
+    }
+    else
+    {
+      realm_manager = new RealmManager(diameter_stack,
+                                       "",
```

Comment:
You think this function is provided by the diameter.conf file - you're going to include this in your testing